raiden/ui/app.py

```diff
@@ -320,6 +320,18 @@ def run_app(
             fg="yellow",
         )
 
+    monitoring_contract_required = (
+        enable_monitoring and CONTRACT_MONITORING_SERVICE not in contracts
+    )
+    if monitoring_contract_required:
+        click.secho(
+            "Monitoring is enabled but the contract for this network was not found. "
```

Comment:
would it be possible to provide the network name here? If not then can you say "ethereum network" instead?

Reply:
I guess since we can't find the network, we don't know it's name. I chose your second suggestion.

Reply:
Thanks man. Please go ahead and merge then.